SU-HSDO/suhumsci
# HSD8-515 color module

# READY FOR REVIEW (Not sure if ready to merge)

# Summary
- Enable color module support to change the color schemes of the website.
- Color schemes would need to be created.

# Needed By (Date)
- some day

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear cache
1. go into the theme settings
1. change the color settings
1. view the home page to validate the change appears.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
